packages/mocha/README.md

```diff
@@ -195,6 +195,24 @@ module.exports = {
 };
 ```
 
+## Webstorm Mocha Runner
+Webstorm has a mocha runner that enables running mocha tests within the IDE. The mocha runner also allows for debugging of mocha tests in Webstorm. Basic setup can be found at [Webstorm's Mocha Run/Debug documentation](https://www.jetbrains.com/help/webstorm/run-debug-configuration-mocha.html). _Note: since neutrino takes care of installing mocha, you do not need to install mocha separately._ Once you are familiar with the basics, the following steps will allow you to use Webstorm's mocha runner with neutrino.
+ 
+- create a `.babelrc` in project root. _As of `neutrino 8.x`, the config provided assumes neutrino is still relying on `babel 6.x`.  If neutrino updates to use `babel 7.x`, then the plugin names below will likely need to be updated to reflect the updated `babel 7.x` version._
```

Comment:
I wanted to use `1.` for numbered list, but I couldn't get markdown to continue the numbered list with the code snippet for `.babelrc`.  So I just resorted to using bullets instead, which is fine for me as well.